shell/platform/windows/settings_plugin_win32.cc

```diff
@@ -26,29 +30,47 @@ SettingsPluginWin32::SettingsPluginWin32(BinaryMessenger* messenger,
     : SettingsPlugin(messenger, task_runner) {
   RegOpenKeyEx(HKEY_CURRENT_USER, kGetPreferredBrightnessRegKey,
                RRF_RT_REG_DWORD, KEY_NOTIFY, &preferred_brightness_reg_hkey_);
+  RegOpenKeyEx(HKEY_CURRENT_USER, kGetTextScaleFactorRegKey, RRF_RT_REG_DWORD,
+               KEY_NOTIFY, &text_scale_factor_reg_hkey_);
 }
 
 SettingsPluginWin32::~SettingsPluginWin32() {
   StopWatching();
   RegCloseKey(preferred_brightness_reg_hkey_);
+  RegCloseKey(text_scale_factor_reg_hkey_);
 }
 
 void SettingsPluginWin32::StartWatching() {
   if (preferred_brightness_reg_hkey_ != NULL) {
     WatchPreferredBrightnessChanged();
   }
+  if (text_scale_factor_reg_hkey_ != NULL) {
```

Comment:
nit: It looks like you're inconsistently using `NULL`/`nullptr`.

Reply:
done